ocp_resources/datavolume.py

```diff
@@ -216,20 +215,64 @@ def scratch_pvc(self):
             client=self.client,
         )
 
+    def _dv_phase(self):
+        if self.exists:
+            return self.instance.status.phase
+        return DV_DELETED_BY_GC
+
     def _check_none_pending_status(self, failure_timeout=120):
         # Avoid waiting for "Succeeded" status if DV's in Pending/None status
         sample = None
         try:
             for sample in TimeoutSampler(
                 wait_timeout=failure_timeout,
                 sleep=10,
-                func=lambda: self.instance.status.phase,
+                func=self._dv_phase,
             ):
                 # If DV status is Pending (or Status is not yet updated) continue to wait, else exit the wait loop
                 if sample in [self.Status.PENDING, None]:
```

Comment:
If dv does not exist - `self._dv_phase` will be True and the function will be existed even if the dv was not created breaking current functionality

Reply:
I check for pvc bound as well, if dv didn't crested we won't have the pvc bound.
this function just check dv is not stuck